site/app/templates/admin/admin_gradeable/AdminGradeablePeers.twig

```diff
@@ -312,17 +312,18 @@
     * @param {boolean} grader_removed true if existing grader was removed
     * @param {String} button_id the ID of the submit button to be enabled / disabled
     * @param {String} error_div_id the ID of the error message to be displayed if submit is disabled
-    * @param {boolean} self_grading true if the student and grader is different
+    * @param {boolean} self_grading true if the student and grader is same
     */
     function updateSubmitButton(valid_grader, valid_students, new_grader, grader_removed, button_id, error_div_id, self_grading) {
-        if (grader_removed || (valid_grader && !valid_students.includes(false) && (!new_grader || valid_students.includes(true)))) {
+        console.log(self_grading);
```

Comment:
Currently a warning is inserted before the webpage. This log is possibly the cause.
![Error](https://github.com/user-attachments/assets/0c22686f-ab22-4e87-997a-88b42e13a3de)